flink-metrics/flink-metrics-core/src/main/java/org/apache/flink/metrics/CommonMetricNames.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.metrics;
+
+import org.apache.flink.annotation.PublicEvolving;
+
+/** Collection of metric names. */
+@PublicEvolving
+public class CommonMetricNames {
+    private CommonMetricNames() {}
+
+    public static final String SUFFIX_RATE = "PerSecond";
+
+    public static final String IO_NUM_RECORDS_IN = "numRecordsIn";
```

Comment:
Is this really necessary? TBH I find it a bit questionable because the field becomes part of the API, and not the actual value within (which is what we actually care about).